controllers/autoscaling.go

```diff
@@ -218,7 +218,7 @@ func (r *HorizontalRunnerAutoscalerReconciler) calculateReplicasByPercentageRunn
 	var desiredReplicas int
 	fractionBusy := float64(numRunnersBusy) / float64(numRunners)
 	if fractionBusy >= scaleUpThreshold {
-		scaleUpReplicas := int(float64(numRunners)*scaleUpFactor + 0.5)
+		scaleUpReplicas := int(float64(numRunners) * scaleUpFactor)
```

Comment:
> Removing 0.5 used to round up any scale-up calculations to the next highest integer. This can also be accomplished by re-setting the scale-up factor.

Ah! That makes sense. Can we use `math.Ceil(float64(numRunners) * scaleUpFactor)` instead?